src/client/src/components/abstract/pieChart/logic.js

```diff
@@ -0,0 +1,63 @@
+import * as d3 from "d3";
+export const colors = [
```

Comment:
@knanahassouna1 
As a future improvement, it might be nice if parent components could pass a colors prop into the pieChart component (and then it would pass the colors into makePie here).  That would make it easier for a parent component to coordinate colors between its pie chart and another piece of its UI (for example, a list with many rows of participants displayed next to a pie chart also containing those participants).  The colors prop could even be optional, and it could use this hard-coded list of colors as a default value.

Reply:
Thanks tom , but i and me team discuss the color schema and we choose this schema to all project , but i like your idea